analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -60,11 +60,17 @@ public class SymbolReferenceAnalyzerTest
         public void Verify_Method_PreciseLocation_VB(ProjectType projectType) =>
             Verify("Method.vb", projectType, references =>
             {
-                references.Select(x => x.Declaration.StartLine).Should().BeEquivalentTo(1, 3, 6);   // class 'Sample' on line 1, method 'Method' on line 3, method 'Go' on line 6
-                var methodDeclaration = references.Single(x => x.Declaration.StartLine == 3);
-                methodDeclaration.Declaration.Should().BeEquivalentTo(new TextRange { StartLine = 3, EndLine = 3, StartOffset = 15, EndOffset = 21 });
-                methodDeclaration.Reference.Should().HaveCount(1);
-                methodDeclaration.Reference.Single().Should().BeEquivalentTo(new TextRange { StartLine = 7, EndLine = 7, StartOffset = 8, EndOffset = 14 });
+                references.Select(x => x.Declaration.StartLine).Should().BeEquivalentTo(1, 3, 6, 10);   // class 'Sample' on line 1, method 'Method' on line 3, method 'Go' on line 6
```

Comment:
The comment is out of date now. Line 6 doesn't contain `Go` anymore and line 10 is not documented.

This turned out to be a bit overkill (that I didn't foresee). The version with extra file was probably easier, as we don't need to assert all of this stuff twice.